backend/src/test/java/ch/puzzle/okr/controller/ObjectiveControllerIT.java

```diff
@@ -186,4 +196,23 @@ void throwExceptionWhenObjectiveWithIdCantBeFoundWhileDeleting() throws Exceptio
         mvc.perform(delete("/api/v2/objectives/1000").with(SecurityMockMvcRequestPostProcessors.csrf()))
                 .andExpect(MockMvcResultMatchers.status().isNotFound());
     }
+
+    @Test
+    void shouldReturnIsCreatedWhenObjectiveWasDuplicated() throws Exception {
+        BDDMockito.given(objectiveAuthorizationService.getBusinessService()).willReturn(objectiveBusinessService);
+        BDDMockito.given(objectiveAuthorizationService.getAuthorizationService()).willReturn(authorizationService);
+        BDDMockito.given((objectiveBusinessService).duplicateObjective(anyLong(), any(), any())).willReturn(objective1);
+        BDDMockito.given(objectiveMapper.toDto(objective1)).willReturn(objective1Dto);
+        BDDMockito.given(authorizationService.getAuthorizationUser()).willReturn(authorizationUser);
+
+        mvc.perform(post("/api/v2/objectives/" + objective1.getId()).contentType(MediaType.APPLICATION_JSON)
```

Comment:
Use https://www.baeldung.com/integration-testing-in-spring#3-send-get-request-with-path-variable